src/Entity/User.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\UserRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
+
+#[ORM\Entity(repositoryClass: UserRepository::class)]
+#[UniqueEntity('email')]
+class User
+{
+    //ATTRIBUTES
+        //id
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private ?int $id = null;
+
+        //firstname
+    #[ORM\Column(type: 'string', length: 255)]
+    private string $firstname;
+
+        //name
+    #[ORM\Column(type: 'string', length: 255)]
+    private string $name;
+
+        //email
+    #[ORM\Column(type: 'string', length: 255, unique: true)]
+    private string $email;
+
+        //password
+    #[ORM\Column(type: 'text')]
+    private string $password;
+
+        //status
+    #[ORM\Column(type: 'string', length: 255)]
+    private string $status;
+
+    #[ORM\OneToMany(mappedBy: 'userId', targetEntity: Chat::class, orphanRemoval: true)]
+    /**
+     * @var Collection<int, Chat>
+     */
+    private Collection $chats;
+
+    //CONSTRUCTOR
+    public function __construct(string $firstname,string $name, string $email, string $password)
+    {
+        $this->firstname = $firstname;
+        $this->name = $name;
+        $this->email = $email;
+        $this->password = $password;
+        $this->status = 'USER';
```

Comment:
You should put this king of value in a const.
```suggestion
        $this->status = User::USER';
```